modules/plugin/mongodb/src/main/java/org/geotools/data/mongodb/FilterToMongo.java

```diff
@@ -211,8 +212,41 @@ public Object visit(Or filter, Object extraData) {
     @Override
     public Object visit(Not filter, Object extraData) {
```

Comment:
For reference:
https://docs.mongodb.com/manual/reference/operator/query/not/#not